### DISCUSSION TITLE
Orchard Harvest online on the 31st of May

### DISCUSSION THREAD

**Discussion by viktoriamagyar** [OrchardCMS/OrchardCore#13574] [General]
Dear Orchard community members!
As many of you know, we have been working hard over the past months to organize the next Orchard Harvest in Las Vegas after the last Harvest in 2017. Unfortunately, based on the current economic situation and personal feedback, Las Vegas won’t work this year. Not to mention that unfortunately Microsoft Build, announced after we decided on our dates, also falls in line with the original dates we chose. So a date change was also necessary.

However, we did not want to let go of this opportunity to meet completely. Therefore, I am pleased to inform you that we will be holding our first **online Orchard Harvest** starting on the **Wednesday 31st of May at 13:00 UTC.** You can start your registration for the event right now: https://events.teams.microsoft.com/event/bfb311dc-5fb0-42d7-bcaf-b8b248f344bd@959ede5a-38e9-44e6-adc8-85553beb8220

Of course, we are still looking for speakers. You can apply to present by 12 May at the following link: https://forms.office.com/e/pfiExtEUuZ
We aim to provide feedback on these within a week. We are expecting presentations of about 45 minutes with 6-8 presentations.
(Besides the presentations, you will have the opportunity to get to know each other a bit better in break-out rooms and to network.)

We are very much looking forward to returning to face-to-face meetings at a later date, but in the meantime please take advantage of this opportunity and join as many of you as possible! We’ll have a day to learn and mingle. :)

**Comment by hishamco** [comment]
Thanks for the work that you did @viktoriamagyar, I have a question about the time, did you consider a proper time for people that might attend from different time zones

**Reply by viktoriamagyar** [reply]
@hishamco Thank you and thank you for the question :) 
Yes, I was trying to find a time that could suit most people.
